workflows/theiacov/wf_theiacov_illumina_se.wdl

```diff
@@ -59,6 +59,9 @@ workflow theiacov_illumina_se {
     String? pangolin_docker_image
     # qc check parameters
     File? qc_check_table
+    # Kraken parameters
+    String? target_organism
+    File kraken_db = "gs://theiagen-large-public-files-rp/terra/databases/kraken2/kraken2_humanGRCh38_viralRefSeq_20240828.tar.gz"
```

Comment:
should this be kraken2 db?

how far are we taking the standardization? if we're doing everything, we need to update everything

Reply:
Agreed, and changed!